patchkit-launcher-qt-src/src/locations.cpp

```diff
@@ -127,7 +132,7 @@ QString Locations::patcherDirectoryPath() const
 
 QString Locations::patcherInstallationInfoFilePath() const
 {
-    return QDir::cleanPath(patcherDirectoryPath() + "/" + Config::patcherDirectoryName);
+    return QDir::cleanPath(patcherDirectoryPath() + "/" + Config::patcherInstallationInfoFileName);
```

Comment:
I've researched the code and found some places that highly depend on file located under that path. Please make a modification that would support previous file location.

Reply:
I disagree, while backwards compatibility with previous Patcher versions is required, backwards compatibility with previous Launcher versions is not, as users won't typically update their launchers.

Reply:
Ok, I agree with that. @genail please share your opinion about it, I'd like to know what do you think about it.

Reply:
I also agree.